samples/tfm_integration/tfm_psa_test/sample.yaml

```diff
@@ -0,0 +1,19 @@
+common:
+    tags: tfm
+    platform_allow: mps2_an521_nonsecure v2m_musca_s1_nonsecure
+                    nrf5340dk_nrf5340_cpuappns nrf9160dk_nrf9160ns
+    harness: console
+    harness_config:
+      type: multi_line
+      regex:
+        - "\\*\\*\\*\\*\\* PSA Architecture Test Suite - Version .* \\*\\*\\*\\*\\*"
+        - "TOTAL FAILED *: 0"
+sample:
+  name: "TFM PSA Test"
+
+tests:
```

Comment:
Unless the tests are currently broken, adding CRYPTO and INITIAL_ATTESTATION as tests as well would make it possible to execute them in CI right away